src/hooks/useDistributionUpdate.ts

```diff
@@ -0,0 +1,12 @@
+import { ManageDistributionFormUpdate } from '../pages/Admin/types';
+import axiosClient from '../utils/axios/axios';
+import { DistributionUpdateResult } from './types/DistributionUpdateResult';
+
+export const UseDistributionUpdate = async (
+  data: ManageDistributionFormUpdate
+): Promise<DistributionUpdateResult> => {
+  return await axiosClient.post(
+    'http://localhost:4000/export-unit-administrative/distribution-update',
+    data
+  );
+};
```

Comment:
Đây không phải là cách viết hook

```suggestion
export const useDistributionUpdate = () => {
    const mutation = useCallback((data: ManageDistributionFormUpdate): Promise<
        DistributionUpdateResult
    > => {
        return axiosClient.post(
            'http://localhost:4000/export-unit-administrative/distribution-update',
            data
        );
    }, []);
    return [mutation];
}
```